src/config/search-page-config.interface.ts

```diff
@@ -8,5 +8,9 @@ export interface SearchConfig extends Config {
      * Used by {@link UploadBitstreamComponent}.
      */
     advancedFilters: AdvancedSearchConfig;
+  /**
+   * Number used to render n skeletons while the filters are loading
```

Comment:
I'm not sure I understand this description, which means other developers may not.  What are "skeletons" and what do they have to do with "defaultFilterCount"?

Is there a different way we can describe this setting?  Or maybe given an example of what will happen if you set this to 5 or 0 or 10.